test/PushNotificationTests/APITests.cpp

```diff
@@ -116,6 +141,25 @@ namespace Test::PushNotifications
             VERIFY_ARE_EQUAL(exitCode, 0);
         }
 
+        const std::wstring GetDeploymentDir()
+        {
+            WEX::Common::String deploymentDir;
+            WEX::TestExecution::RuntimeParameters::TryGetValue(L"TestDeploymentDir", deploymentDir);
+            return reinterpret_cast<PCWSTR>(deploymentDir.GetBuffer());
+        }
+
+        void RunTestUnpackaged(const PCWSTR& testName, const int& waitTime)
```

Comment:
You can do this via TAEF. Worst case define 2 tests, one with RunAs:UWP and one without.

No need to roll your own test process launcher